deployment/arm/deploy-global.sh

```diff
@@ -1,23 +1,36 @@
 #!/bin/bash
+set -e
 parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
 cd "$parent_path"
 
 businessUnit=$1
 appName=$2
 env=$3
+primaryRegion=$4
+secondaryRegion=$5
+tertiaryRegion=$6
+
+if [[ -z $businessUnit || -z $appName || -z $env || -z $primaryRegion || -z $secondaryRegion || -z $tertiaryRegion ]]; then
+  echo 'One or more variables are undefined'
+  exit 1
+fi
+
 # Regions are passed in as additional arguments
-scope="$businessUnit-$appName-$env-gbl"
+scope="$businessUnit-$appName-$env-glb"
```

Comment:
I think the abbreviation for global is typically `gbl` no?  Did you see different guidance somewhere?

Reply:
I`ll change it back to gbl. I misread it :)